Source/Tools/FEXLoader/LinuxSyscalls/Syscalls/Thread.cpp

```diff
@@ -292,6 +292,11 @@ namespace FEX::HLE {
     REGISTER_SYSCALL_IMPL_FLAGS(exit, SyscallFlags::OPTIMIZETHROUGH | SyscallFlags::NOSYNCSTATEONENTRY | SyscallFlags::NORETURN,
       [](FEXCore::Core::CpuStateFrame *Frame, int status) -> uint64_t {
       auto Thread = Frame->Thread;
+
+      // TLS/DTV teardown is something FEX can't control. Disable glibc checking when we leave a pthread.
+      // Since this thread is hard stopping, we can't track this in FEX's thread handling.
```

Comment:
> we can't track this

What is this referring to?

Reply:
Changed the comment that it is about the TLS/DTV teardown.